.travis.yml

```diff
@@ -16,8 +16,8 @@ env:
 before_install:
 - echo ${DOCKER_PASSWORD} | docker login -u=decaftravis --password-stdin
```

Comment:
Untested but something like the following should be added in the `before_install` section:
```yaml
  - docker build -t ${IMAGE_REPO}:update .
  - docker run --rm --volume ".:/home/kaa/app" --user "kaa" ${IMAGE_REPO}:update pipenv lock
```

Reply:
I think the `Pipfile.lock` should then be removed from the repo.

Reply:
My bad, `docker run --rm --volume ".:/home/kaa/app" ...` only works in docker-compose. It should be `docker run --rm --volume "${PWD}:/home/kaa/app" ...`